rdr_common/code_constants.py

```diff
@@ -79,6 +79,8 @@
                  '1PXR2']
 BIOBANK_TESTS_SET = frozenset(BIOBANK_TESTS)
 
+BIOBANK_DNA_TESTS = ['1ED10', '2ED10', '1ED04', '1SAL', '1SAL2']
```

Comment:
This is already in our config, here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/config/base_config.json#L32

We should avoid duplicating this in constants if possible.